src/document/position.js

```diff
@@ -97,13 +119,13 @@ CKEDITOR.define( [ 'utils', 'ckeditorerror' ], ( utils, CKEditorError ) => {
 		}
 
 		/**
-		 * Two positions equal if paths are equal.
+		 * Two positions equal if paths are equal and roots are the same.
 		 *
 		 * @param {document.Position} otherPosition Position to compare.
 		 * @returns {Boolean} True if positions equal.
 		 */
 		isEqual( otherPosition ) {
```

Comment:
One more thing - I am not sure if `isEqual` is a better name than `isSame` but since we also have `isEqual` methods in other classes I'd just stick with this name to make it unified across classes.